queries/fetch_remaining_transactions.sql

```diff
@@ -0,0 +1,16 @@
+SELECT 
```

Comment:
This query doesn't work (it has syntax errors).

Also, what is the data you need from the other confirmed transactions in the block for milestone 2?